modules/tinymce/src/plugins/table/test/ts/browser/TabKeyNavigationTest.ts

```diff
@@ -16,30 +18,51 @@ describe('browser.tinymce.plugins.table.TabKeyNavigationTest', () => {
     },
     base_url: '/project/tinymce/js/tinymce'
   }, [ Plugin, Theme ], true);
+  let events: Array<EditorEvent<TableModifiedEvent>> = [];
+  const logEvent = (event: EditorEvent<TableModifiedEvent>) => {
+    events.push(event);
+  };
+  afterEach(() => events = []);
```

Comment:
Please make sure to add relevant whitespace for readability. We also should try and keep consistency here, so copying what's been used elsewhere:
```suggestion

  let events: Array<EditorEvent<TableModifiedEvent>> = [];
  const logEvent = (event: EditorEvent<TableModifiedEvent>) => {
    events.push(event);
  };
  const clearEvents = () => events = [];

  afterEach(() => clearEvents());
```